maintenance/projects/js-toolkit/packages/liferay-npm-bundler/src/dependencies.js

```diff
@@ -57,12 +57,15 @@ export function addPackageDependencies(
 	dependencies = dependencies.concat(extraDependencies);
 
 	const dependencyDirs = dependencies
-		.map((dependency) => {
-			return resolveDependencyDir(basedirPath, packageJson, dependency);
-		})
-		.filter((dependencyDir) => {
-			return dependencyDir != null;
-		});
+		.filter(
+			(dependency) =>
+				!project.presetInfo.isAutopreset ||
+				dependency !== project.presetInfo.name
+		)
+		.map((dependency) =>
+			resolveDependencyDir(basedirPath, packageJson, dependency)
```

Comment:
Shouldn't the name be `baseDirPath` instead of `basedirPath`. I know it's preexisting code, but just to bring attention to it.

Reply:
Yeah, makes sense to use uppercase in word stops. This is probably inconsistent, like [basename](https://nodejs.org/api/path.html#path_path_basename_path_ext) or [dirname](https://nodejs.org/api/path.html#path_path_dirname_path)...

Reply:
Addressed in https://github.com/liferay/liferay-frontend-projects/pull/576